examples/local/README.md

```diff
@@ -0,0 +1,91 @@
+# Running cert-operator Locally
+
+**Note:** This should be used only for testing end development. See the
+[/kubernetes/][kubernetes-dir] directory and [Secrets][secrets-doc] for
+a production ready configuration.
+
+[kubernetes-dir]: https://github.com/giantswarm/cert-operator/tree/master/kubernetes
+[secrests-doc]: https://github.com/giantswarm/cert-operator#secrets
+
+This guide explains how to get running cert-operator locally. For example on
+minikube. Certificates created here are meant to be used by [aws-operator].
+
+All commands are assumed to be run from `examples/local` directory.
+
+[aws-operator]: https://github.com/giantswarm/aws-operator
+
+## Preparing Templates
+
+```bash
+for f in *.tmpl.yaml; do
+    sed \
+        -e 's/${CLUSTER_NAME}/example-cluster/g' \
+        -e 's/${COMMON_DOMAIN}/company.com/g' \
+        -e 's/${VAULT_HOST}/vault/g' \
+        -e 's/${VAULT_TOKEN}/secret_sauce/g' \
+        ./$f > ./${f%.tmpl.yaml}.yaml
+done
```

Comment:
I think this is handy to do it like this. All placeholders are replaced and you can just follow the guide. Also it's easy to do the diff and see what changed.
All `.yaml` files in examples/local directory are ignored, so it's easy to play with it.